### ISSUE TITLE
Improve error message when keyframes are invalid

### ISSUE THREAD

**Issue by GuYounes** [ampproject/amphtml#20971]
## What's the issue?

Cannot use TranslateY in animation

## How do we reproduce the issue?

I have an animation translate-Y which is supposed to translate an element : 

![image](https://user-images.githubusercontent.com/32415454/53126540-a998b280-3560-11e9-9b59-16e705fa2276.png)

and I call it this way : 

![image](https://user-images.githubusercontent.com/32415454/53126614-d0ef7f80-3560-11e9-9295-c88ad40313ff.png)

But then I have a JS error in the console :

![image](https://user-images.githubusercontent.com/32415454/53126657-eebce480-3560-11e9-8bc4-a251a3d7f5a8.png)

I tried to inspect a little bit, and I found that in web-animation.js, in the init function : 

![image](https://user-images.githubusercontent.com/32415454/53126952-9afecb00-3561-11e9-9c2c-58ce9a4f198e.png)
 

## What browsers are affected?

I experienced this issue on Chrome 71, Windows 10

## Which AMP version is affected?

Version 1902151859190

**Comment by sparhami** [comment]
I'm not sure that is the cause of the error, but you need units for your translateY (e.g. `-68px` or `-68%` and not `-68`).

**Comment by GuYounes** [comment]
Indeed ,sorry for that !
Thanks

**Comment by sparhami** [comment]
Reopening the issue for improving the error message, it should say something better than "Partial keyframes are not supported.

**Comment by torch2424** [comment]
@sparhami I assigned this to you since you seem to have context, feel free to un-assign yourself 😄

**Comment by alanorozco** [comment]
This error message [comes directly from the platform](https://source.chromium.org/chromium/chromium/src/+/master:third_party/blink/renderer/core/animation/effect_input.cc;l=718?q=%22partial%20keyframes%20are%20not%20supported%22&ss=chromium&originalUrl=https:%2F%2Fcs.chromium.org%2F) and is [reflected by the polyfill as well](https://github.com/web-animations/web-animations-js/blob/3f9af6bfacf642b7f8cd28d3c8128ff8e80ccb9a/src/keyframe-interpolations.js#L63).

If we believe the error can be written in a better way, we should specify that to the Chrome/other platforms teams. Otherwise we'd have to special case this and inspect the effect spec, which would be cumbersome.
